CPS-????/README.md

```diff
@@ -0,0 +1,137 @@
+---
+CPS: 24
+Title: Canonical CBOR Serialization Standard
+Category: Tools
+Status: Open
+Authors:
+    - Hinson Wong <wongkahinhinson@gmail.com>
+    - Tsz Wai Wu <tszwaiwu.96@gmail.com>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/1109
+Created: 2025-10-29
+License: CC-BY-4.0
+---
+
+## Abstract
+
+There is no canonical CBOR serialization standard in Cardano. While this is a delibrate design choice initially, standardizing it has growing popularity in Cardano developer community as evidenced by developer meetups such as Cardano Builder Fest 2025 hosted in Vietnam. This CPS outlines the motivation of the growing concern of fragmented CBOR serialization patterns across in the community.
+
+## Problem
+
+<!-- A more elaborate description of the problem and its context. This section should explain what motivates the writing of the CPS document. -->
+
+The Cardano ledger accepts any valid CBOR encoding for transactions and on-chain data. While this flexibility was intentional to encourage ecosystem diversity, it has created significant interoperability challenges as the tooling landscape has matured. The same logical data can be encoded in multiple ways (map key ordering, integer encoding, definite vs. indefinite length, etc.), leading to different byte representations and transaction hashes.
+
+### Core Issues
+
+**Transaction Hash Instability**: When a transaction is passed between tools or wallets for signing, each may re-serialize it differently. Since transaction hashes are computed over CBOR bytes, logically identical transactions produce different hashes. This breaks:
+
+- Multi-signature workflows where each signer's wallet may re-serialize the transaction
+- Cross-tool transaction building where fee calculations depend on exact byte size
```

Comment:
I don't quite follow this one. If there is any change to a transaction body then the bytes might change which could affect the fee. That is quite normal and expected.

If you mean that some tooling might want to change some part of the transaction body that it itself should not change the size if canonical CBOR was used (eg. change a required signer, i.e. swap one hash for another), then I can see it as an argument, but I don't understand why such tooling couldn't just recompute the minimum fee?